docs/modeling-your-data/running-data-models-via-snowplow-bdp/dbt/using-dbt/index.md

```diff
@@ -0,0 +1,179 @@
+
+<Tabs groupId="dbt-packages">
+<TabItem value="redshift" label="Redshift" default>
+
+See [dbt connection setup](https://docs.getdbt.com/reference/warehouse-setups/redshift-setup) for more information.
+
+```yaml
+profile_name:
+  outputs:
+    prod:
+      type: redshift
+      threads: [1 or more]
+      host: [hostname.region.redshift.amazonaws.com]
+      port: 5348
+      user: datamodeling
+      pass: "{{ env_var('REDSHIFT_DATAMODELING_PASSWORD') }}"
+      dbname: [database name]
+      schema: [schema name]
+  target: prod
+```
+
+</TabItem>
+<TabItem value="snowflake" label="Snowflake">
+
+See [dbt connection setup](https://docs.getdbt.com/reference/warehouse-setups/snowflake-setup) for more information.  
+[Finding your snowflake account name](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html#non-vps-account-locator-formats-by-cloud-platform-and-region)
+
+```yaml
+profile_name:
+  outputs:
+    prod:
+      type: snowflake
+      account: 
+      user: ANALYTICS_PROD_USER
+      password: "{{ env_var('SNOWFLAKE_DATAMODELING_PASSWORD') }}"
+      role: ANALYTICS_PROD_ROLE
+      database: ANALYTICS_PROD_DB
+      warehouse: ANALYTICS_PROD_WH
+      schema: DBT_PROD
+      threads: [1 or more]
+      client_session_keep_alive: true
+  target: prod
+```
+
+</TabItem>
+<TabItem value="bigquery" label="BigQuery">
+
+See [dbt connection setup](https://docs.getdbt.com/reference/warehouse-setups/bigquery-setup) for more information.
+
+
+```yaml
+profile_name:
+  outputs:
+    prod:
+      type: bigquery
+      method: oauth
+      project: [GCP project id]
+      dataset: [the name of your dbt dataset] # You can also use "schema" here
+      threads: [1 or more]
+      keyfile: "{{ env_var('GOOGLE_APPLICATION_CREDENTIALS') }}" 
+  target: prod
+```
+
+</TabItem>
+<TabItem value="databricks" label="Databricks">
+
+See [dbt connection setup](https://docs.getdbt.com/reference/warehouse-setups/databricks-setup) for more information.
+
+```yaml
+profile_name:
+  outputs:
+    prod:
+      type: databricks
+      catalog: [optional catalog name, if you are using Unity Catalog, is only available in dbt-databricks>=1.1.1]
+      schema: [schema name]
+      host: [yourorg.databrickshost.com]
+      http_path: [/sql/your/http/path]
+      token: "{{ env_var('DATABRICKS_DATAMODELING_TOKEN') }}" # Personal Access Token (PAT)
+      threads: [1 or more]  # optional, default 1
+  target: prod
+```
+
+</TabItem>
+</Tabs>
+
+### 2. The data modeling configuration
+
+Data models can be configured via the Snowplow BDP Console:
+
+![](images/Screenshot-2021-11-15-at-20.15.28.png)
+
+In the first step, you can provide the data model name, a description, and the owner(s) of the data model that will be alerted in case of failure and finally select the engine that should run your datamodel (DBT in this case):
```

Comment:
```suggestion
In the first step, you can provide the data model name, a description, and the owner(s) of the data model that will be alerted in case of failure and finally select the engine that should run your datamodel (dbt in this case):
```

The UI needs fixing (cc @andreamarcolin)

Reply:
Agree, leaving open for UI tag to be agknowledged then can commit suggestion.